typescript/packages/plugins/coinmarketcap/src/parameters.ts

```diff
@@ -0,0 +1,60 @@
+import { createToolParameters } from "@goat-sdk/core";
+import { z } from "zod";
+
+export class CryptocurrencyListingsParameters extends createToolParameters(
+    z.object({
+        start: z.number().optional().describe("Starting position of results"),
+        limit: z.number().optional().describe("Number of results to return"),
+        sort: z.string().optional().describe("What field to sort the list by"),
```

Comment:
Add the options here, default is `market_cap`

"name""symbol""date_added""market_cap""market_cap_strict""price""circulating_supply""total_supply""max_supply""num_market_pairs""volume_24h""percent_change_1h""percent_change_24h""percent_change_7d""market_cap_by_total_supply_strict""volume_7d""volume_30d"